dotcom-rendering/src/web/components/NewslettersBadge.tsx

```diff
@@ -0,0 +1,65 @@
+import { css } from '@emotion/react';
+import { brandAlt, headline, neutral } from '@guardian/source-foundations';
+
+const textStyle = (textColour: string) => css`
+	${headline.small({ fontWeight: 'bold' })};
+	font-size: 19.5px;
+	letter-spacing: -0.5px;
+	fill: ${textColour};
+`;
+
+interface Props {
+	textColour?: string;
+	borderColour?: string;
```

Comment:
We're not actually using these props anywhere (I think?) so I guess we could just inline these values?

If I'm wrong and we are using them, ignore me!

Reply:
We aren't using them now, but I've seen other designs in the pipeline using the same logo, but with different colors (eg  dark text on a light background) - I'd tend to keep the option to future use, but not a strong preference.

Reply:
no problem - I'll inline the values!